nmdc_automation/models/workflow.py

```diff
@@ -101,13 +101,16 @@ class WorkflowConfig:
     # populated after initialization
     children: Set["WorkflowConfig"] = field(default_factory=set)
     parents: Set["WorkflowConfig"] = field(default_factory=set)
-    data_object_types: List[str] = field(default_factory=list)
+    input_data_object_types: List[str] = field(default_factory=list)
 
     def __post_init__(self):
-        """ Initialize the object """
+        """ Parse input data object types from the inputs """
         for _, inp_param in self.inputs.items():
+            # Some input params are boolean values, skip these
+            if isinstance(inp_param, bool):
+                continue
```

Comment:
@mbthornton-lbl  this boolean needs to be pushed to the inp_param, not skipped. See https://github.com/microbiomedata/metaAssembly/blob/master/input.json